.github/workflows/native_jni_s3_pytorch.yml

```diff
@@ -45,7 +48,7 @@ jobs:
           aws-region: us-east-2
       - name: Copy files to S3 with the AWS CLI
         run: |
-          aws s3 cp pytorch/pytorch-native/build/*djl_torch.* s3://djl-ai/publish/pytorch-1.7.1/jnilib/0.10.0/${{ matrix.os }}-x86_64/cpu/
+          aws s3 cp pytorch/pytorch-native/build/${{ matrix.lib }} s3://djl-ai/publish/pytorch-1.7.1/jnilib/0.10.0/${{ matrix.os }}-x86_64/cpu/
```

Comment:
Is this causing proble?

Reply:
the window-cpu actually failed but still show pass 
The user-provided path pytorch/pytorch-native/build/*djl_torch.* does not exist.

Reply:
I see, windows doesn't expend `* ` as linux do. it can only do djl_torch.*